recipes/xapian-core/all/conanfile.py

```diff
@@ -162,7 +162,9 @@ def package_info(self):
         if not self.options.shared:
             if self.settings.os == "Linux":
```

Comment:
I don't know whether there is some other Linux dependent logic in this recipe.
```suggestion
            if self.settings.os in ("Linux", "FreeBSD"):
```

Reply:
no other Linux logic in the recipe